playbooks/roles/streisand-gateway/templates/openssl-local.cnf.j2

```diff
@@ -0,0 +1,85 @@
+distinguished_name = req_distinguished_name
```

Comment:
Is there a reason you put the `distinguished_name` prompt mapping at the toplevel default config? I _think_ this can go into a `[req`] config stanza (or perhaps into the existing `[v3_req]`?).

Reply:
This was in all of the cargo-cult `openssl.cnf`s. I didn't think I could reliably change it.

https://superuser.com/a/947080

https://github.com/stanzgy/wiki/blob/master/network/openssl-self-signed-certs-cheatsheet.md

https://www.phildev.net/ssl/opensslconf.html

It's possible all this cargo is from the same cult. Sigh, OpenSSL is just such a disaster.

Reply:
> It's possible all this cargo is from the same cult. Sigh, OpenSSL is just such a disaster.

:laughing: I use the `openssl.cnf` passed down through generations on my mother's side. It's my most prized family heirloom.